packaging/suse/trento.spec

```diff
@@ -26,17 +26,14 @@ Group:          System/Monitoring
 URL:            https://github.com/trento-project/trento
 Source:         %{name}-%{version}.tar.gz
 Source1:        vendor.tar.gz
-Source2:        node_modules.spec.inc
-Source3:        package.json
-%include        %_sourcedir/node_modules.spec.inc
 ExclusiveArch:  aarch64 x86_64 ppc64le s390x
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildRequires:  golang-packaging
 BuildRequires:  golang(API) = 1.16
-BuildRequires:  npm
-BuildRequires:  local-npm-registry
 Requires:       golang-github-prometheus-node_exporter
 Provides:       %{name} = %{version}-%{release}
+Provides:       trento = %{version}-%{release}
+Obsoletes:      trento < %{version}-%{release}
```

Comment:
We'll probably have to also include `trento-premium` here in the future, although I'm not sure yet. Just noting this down for our future selves.